content/docs/install/plugins.md

```diff
@@ -14,7 +14,7 @@ In order to recognize `.dvc` files as YAML in Vim, you should add:
 
 ```vim
 " DVC
-autocmd! BufNewFile,BufRead Dvcfile,*.dvc setfiletype yaml
+autocmd! BufNewFile,*.dvc setfiletype yaml
```

Comment:
Can someone who uses Vim confirm this though? Maybe @efiop (since I'm about to ping you about something else 😬)

Reply:
@jorgeorpinel Just from looking at it, seems like it is broken and should be:
```
autocmd! BufNewFile,BufRead *.dvc setfiletype yaml
```
but I would actually keep Dvcfile for now, since it might still be present in older repos.

Reply:
Good point, will roll this back for now.